_data/developer.yml

```
@@ -561,7 +561,10 @@ websites:
     img: twilio.png
     tfa:
       - sms
-    doc: https://www.twilio.com/help/faq/twilio-basics/what-happens-when-i-enable-two-factor-authentication
+      - phone
+      - proprietary
+    doc: https://support.twilio.com/hc/en-us/articles/223136307-Enabling-two-factor-authentication-on-your-Twilio-project
```

```suggestion
    doc: https://support.twilio.com/hc/en-us/articles/223136307
```
The shortened link is sufficient (and probably more stable).